### DISCUSSION TITLE
Improve menu responsiveness community website

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#807] [Suggestions]
![image](https://github.com/gameserverapp/Platform/assets/44717221/75bc5212-935f-4334-8e55-3fe70e9756a4)

Displayed here, using tab - the icon is covering the link for my web shop, preventing access to the page. If you click on it, it takes you to the home page. 

Expanding the web page seems to fix it, but not everyone may be savvy enough to do that, or may have smaller monitor resolutions that prevent it. 
![image](https://github.com/gameserverapp/Platform/assets/44717221/38da938c-a958-4c97-8543-bfadb1ac61fb)

**Comment by mvaessen** [comment]
This is a tough one, mostly because the menu behavior is primarily driven by the content (menu items).

Gsa uses so called `breakpoints`, which causes the website to hide / show the menu based on the size of the screen. 
The way i've set it up is that the menu will reduce in size for smaller screens, but at some point the number of menu items (or the length of them combined) will just be too big. 

While not ideal, the easiest fix for this would be to change some of the menu items, so they become shorter. That should build in some space to deal with responsiveness.

I'm going to move this one to the suggestion section, as it is mostly operating as designed. But it's a good idea to preserve it for when we work on a community website update
